web/src/components/planner/PlannerStatistics.vue

```diff
@@ -39,6 +43,34 @@
             <span v-else class="ml-2">Water: <i class="fas fa-infinity" /></span>
           </v-chip>
         </v-card-text>
+        <v-card-text v-show="!hidden" class="text-body-1">
+          <p v-show="helpText" class="mb-4">
+            <i class="fas fa-info-circle mr-2" />Showing all of the world power consumption and generation.
+          </p>
+          <h1 class="text-h5 mb-4">
+            <i class="fas fa-power-off" />
+            <span class="ml-3">Power Consumption and Generation</span>
+          </h1>
+          <span class="ml-2"><b>[Coming soon!!]<br></b></span>
+          <v-chip
+              class="sf-chip yellow"
+              variant="tonal"
+            >
+              <i class="fas fa-bolt" />
+              <span class="ml-2">
+                [n] MW consumed
+              </span>
+            </v-chip>
+            <v-chip
+              class="sf-chip green"
+              variant="tonal"
+            >
+              <i class="fas fa-wind-turbine" />
```

Comment:
Is Solar Panel clearer? I can't really tell what the wind turbine icon is. 

```suggestion
              <i class="fas fa-solar-panel" />
```